openmdao/drivers/tests/test_scipy_optimizer.py

```diff
@@ -345,6 +345,34 @@ def test_unsupported_equality(self):
 
         self.assertEqual(exception.args[0], msg)
 
+    def test_scipy_missing_objective(self):
+
+        prob = Problem()
+        model = prob.model = Group()
+
+        model.add_subsystem('x', IndepVarComp('x', 2.0), promotes=['*'])
+        model.add_subsystem('f_x', Paraboloid(), promotes=['*'])
+
+        prob.driver = ScipyOptimizeDriver()
+        prob.driver.options['optimizer'] = 'SLSQP'
+
+        prob.model.add_design_var('x', lower=0)
+        prob.model.add_constraint('x', lower=0)
+
+        prob.setup(check=False)
+
+
+        with self.assertRaises(Exception) as raises_msg:
+            prob.run_driver()
+
+        exception = raises_msg.exception
+
+        msg = "ScipyOptimizeDriver requires objective to be declared"
+
+        self.assertEqual(exception.args[0], msg)
+
```

Comment:
Some extra carriage returns in these tests. 2 blank lines should only be at the start of a new class.

Reply:
Fixed in latest push